<!DOCTYPE HTML>
<html>


<!-- Mirrored from ieeebibm.org/BIBM2024/KeynoteSpeech.htm by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 26 Dec 2025 11:14:08 GMT -->
<head>
  <title>2022 IEEE International Conference on Bioinformatics and Biomedicine</title>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="description"
    content="This websites is used to present the content of 2022 IEEE International Conference on Big Data" />
  <meta name="keywords" content="Big Data, IEEE International Conference" />

  <script src="js/jquery.min.html"></script>
  <script src="js/jquery.scrollzer.min.html"></script>
  <script src="js/jquery.scrolly.min.html"></script>
  <script src="js/skel.min.html"></script>
  <script src="js/skel-layers.min.html"></script>
  <script src="js/init.html"></script>


  <link rel="stylesheet" href="style.css">
</head>

<body>

  <h2> Keynote Speakers</h2>

  <br id="main">
  <section id="keynotespeech">
    <div class="container">

      <div id="speechers">
        <table>
          <tr>
            <td width=240px>
              <div align="center" width=240px>
                <img src="image/Dr.%20Sameer%20Antani.jpg" class="auto-style1" />
              </div>
            </td>

            <td width=240px>
              <div align="center" width=240px>
                <img src="image/Shihua%20Zhang.jpg" class="auto-style1" />
              </div>
            </td>
            <td width=240px>
              <div align="center" width=240px>
                <img src="image/Prof%20Pietro%20Li%c3%b2.jpg" class="auto-style1" />
              </div>
            </td>
          </tr>
          <tr>
            <td width=240px>
              <div align="center" width=240px>
                <strong>Sameer Antani, PhD</strong><br /> Principal Investigator, DIR/NLM/NIH<br />
              </div>
            </td>

            <td width=240px>
              <div align="center" width=240px>
                <p><strong>Shihua Zhang</strong><br /> Academy of Mathematics and Systems Science, Chinese Academy
                  of Sciences<br /></p>
              </div>
            </td>

            <td width=240px>
              <div align="center" width=240px>
                <p><strong>Prof Pietro Liò</strong><br />Professor of Computational Biology,  University of Cambridge<br /></p>
              </div>
            </td>
            
          </tr>

          
        </table>
      </div>
    </div>
  </section>



  <h3>Title: Impact of Data Characteristics on Medical Imaging AI Performance: Insights from Recent Research</h3>
  <section id="keynote2">
    <div class="container">
      <header class="major">

        <p><strong>Abstract:</strong> </p>
        <p>Medical imaging artificial intelligence (AI) and machine learning (ML) applications present opportunities
          and significant promise for enhancing disease screening, diagnosis, and overall patient care. These
          advanced algorithms have the potential to revolutionize the medical field by providing more accurate and
          efficient diagnostic tools. However, impressive performance, effectiveness and reliability of these
          algorithms are profoundly influenced by the characteristics of the underlying data that they are trained
          on and operate upon. In this talk, I will delve into the various data characteristics, viz., Volume,
          Variety, Validity, Variety, and Velocity, that play a crucial role in shaping the performance and
          evolution of machine learning models in medical imaging and computer vision. I will draw on experiences
          from my research into screening, diagnostics, and treatment of various cancers, heart disease, and
          infectious diseases such as tuberculosis, and COVID-19 to explore how these characteristics impact AI/ML
          performance and reliability.</p>

        <p><b>Dr. Sameer Antani</b> is a Principal Investigator in the Division of Intramural Research of the
          National Library of Medicine (NLM) at the National Institutes of Health (NIH), USA. His research focuses
          on novel algorithms in medical imaging and multimodal machine learning (ML) and artificial intelligence
          (AI), ML/AI for resource-limited and global health settings, and characterizing data and innovations in
          ML/AI algorithm design for reliable AI predictions with applications in disease screening, diagnostics,
          risk prediction, and treatment. Dr. Antani is a Fellow of the American Institute for Medical and
          Biological Engineering (AIMBE), Fellow of Institute of Electrical and Electronics Engineers (IEEE), and
          Senior Member of the SPIE. He earned his Ph.D. and M.Eng. in Computer Science and Engineering from the
          Pennsylvania State University and his B.E. (First Class with Distinction) in Computer Engineering from the
          Savitribai Phule Pune University (formerly University of Pune), India. He is a recipient of several awards
          for his scientific contributions and research leadership.</p>
      </header>
    </div>
  </section>
  </br>
  </br>




  <h3>Title: Intelligent spatial transcriptomics: methods and applications</h3>
  <section id="keynote2">
    <div class="container">
      <header class="major">

        <p><strong>Abstract:</strong> </p>
        <p>Technological advances in spatial transcriptomics are critical for better understanding the structures
          and functions of tissues in biological research. The combination of intelligent or statistical algorithms
          and spatial transcriptomics has emerged to pave the way for deciphering tissue architecture. We have made
          great efforts to advance intelligent spatial transcriptomics and developed a group of STA- tools.
          Specifically, we first created a graph attention auto-encoder tool STAGATE to identify spatial domains by
          learning low-dimensional latent embeddings via integrating spatial information and gene expression
          profiles. Second, we introduced STAligner for integrating and aligning ST datasets across different
          conditions, technologies, and developmental stages to enable spatially-aware data integration,
          simultaneous spatial domain identification, and downstream comparative analysis. Third, we designed
          STAMarker for identifying spatially domain-specific variable genes with saliency maps in deep learning.
          Fourth, we developed a spatial location-supervised auto-encoder generator STAGE for generating
          high-density spatial transcriptomics. Fifth, we developed STASCAN for deciphering fine-resolution
          cell-distribution maps in spatial transcriptomics. We also extended these tools for biological
          applications in diverse scenarios.</p>

        <p><b>Shihua Zhang</b> received a Ph.D. degree in applied mathematics and bioinformatics from the Academy of
          Mathematics and Systems Science, Chinese Academy of Sciences in 2008. He joined the same institute as an
          Assistant Professor in 2008 and is currently Professor. His research interests are mainly in
          bioinformatics and computational biology, machine learning, and deep learning. His main papers have been
          published in top journals including Cell, National Computational Biology, Nature Communications, Advanced
          Science, Genome Biology, Genome Research, National Science Review, Cell Reports, Nucleic Acids Research,
          JMLR, IEEE TPAMI, IEEE TKDE, IEEE TNNLS, etc. His papers have been selected as “Top Ten Bioinformatics
          Advances in China” in 2021 and 2022, respectively. He won the Youth Science and Technology Award of China,
          Ten Thousand Talent Program—Leading Talents (in Scientific and Technological Innovation) and Young
          Top-notch Talent; NSFC Excellent Youth Fund, CAS Outstanding Young Investigator Program, Lu Jia-Xi Young
          Investigator Award, Zhongchuang Software Talent Award, National Excellent Ph.D. Thesis Award and so on.
          Now he serves as a Section Editor of PLOS Computational Biology and an Editor for Genomics, Proteomics &
          Bioinformatics, etc.</p>
      </header>
    </div>
  </section>

  <h3>Title:  AI and Bioinformatics in Personalised Medicine</h3>
  <section id="keynote2">
    <div class="container">
      <header class="major">

        <p><strong>Abstract:</strong> </p>
        <p>In this talk, I will focus on how AI can support various clinical areas, both at the benchside and bedside levels and considering personalised and population medicine.
          I will then describe a model of a digital patient twin using graph and hypergraph representation learning and considering physiological, clinical and molecular variables (multi-omics and genetics).
           Then, I will discuss the impact of diffusion models in drug discovery.
          Finally, I will discuss explainability methodologies that favour a productive interaction between clinicians and AI.</p>

      </header>
    </div>
  </section>


</body>


<!-- Mirrored from ieeebibm.org/BIBM2024/KeynoteSpeech.htm by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 26 Dec 2025 11:14:10 GMT -->
</html>